climateconnect/climateconnect
# Fix: Preserve and Apply Filters

## Description

Closes #1138

This PR aims to make filters applied to BrowseContent carry over between tabs and apply. 

## Test plan

1. Go to browse page
2. Select some filters
3. Switch tab
4. Delete filters
5. Result should be that filters apply as expect and visible filters are actually applying

## Before landing

1. PR has meaningful title
1. `yarn lint` passes (frontend)
1. `yarn format` passes (frontend)
1. `make format` passes (backend)
